raiden/tests/integration/long_running/test_settlement.py

```diff
@@ -649,9 +648,9 @@ def run_test_start_end_attack(token_addresses, raiden_chain, deposit):
     app2.raiden.chain.wait_until_block(target_block_number=attack_transfer.lock.expiration - 1)
 
     # since the attacker knows the secret he can net the lock
-    attack_channel.netting_channel.unlock(
-        UnlockProofState(unlock_proof, attack_transfer.lock, secret)
-    )
+    # attack_channel.netting_channel.unlock(
+    #     UnlockProofState(unlock_proof, attack_transfer.lock, secret)
+    # )
```

Comment:
These comments look like leftovers?

Reply:
yep, the test is a `xfail`, and instead of deleting @konradkonrad left it so that we can fix it in the future. This is just commented out as a reference for anyone that gets this to fix.

Reply:
In that case, a comment stating this is a good idea.